code/lib/manager-api/src/modules/layout.ts

```diff
@@ -253,7 +253,7 @@ export const init: ModuleFn = ({ store, provider, singleStory, fullAPI }) => {
     },
   };
 
-  const persisted = pick(store.getState(), 'layout', 'ui', 'selectedPanel');
+  const persisted = pick(store.getState(), 'layout', 'selectedPanel');
```

Comment:
Can you explain this change? Seems sort of wide ranging?

Reply:
https://github.com/storybookjs/storybook/blob/42fae30325bb7635e38689e8cc9ece3740d49746/code/lib/types/src/modules/api.ts#L105-L109

According to the types, these are the only options allowed in this object... And I think none of these should really be persisted!